scripts/commit-filter-check.sh

```diff
@@ -59,7 +59,7 @@ commit_message_check (){
             echo $gitmessage
             echo " "
             rm shafile.txt >/dev/null 2>&1
-            set -o errexit
+            exit 1
```

Comment:
Doesn't match the circle ci docs https://circleci.com/docs/2.0/using-shell-scripts/#set-error-flags

Reply:
Have no issue with the change, will try and verify

Reply:
Thank you 👍